src/components/MenuButton/MenuButton.tsx

```diff
@@ -144,4 +145,31 @@ export default class MenuButton extends React.Component<MenuButtonProps> {
   private getMenuItemContent(props: IContextualMenuItemProps) {
     return <MenuButtonItem {...props} />;
   }
+
+  private getStyles = (): IButtonStyles => {
+    return {
+      root: {
+        border: 0,
+        padding: 0,
+        opacity: 0.7,
+        width: `${this.props.iconSize}px`,
+        height: `${this.props.iconSize}px`,
+        color: '#495361',
+      },
+      rootHovered: {
+        backgroundColor: 'transparent',
+        color: '#386cbb',
+        opacity: 1,
+      },
+      rootPressed: {
+        backgroundColor: 'transparent',
+        color: '#386cbb',
+        opacity: 1,
+      },
+      rootExpanded: {
+        color: '#386cbb',
+        opacity: 1,
+      },
+    };
+  };
```

Comment:
You will be generating a new object every time this function gets called, which means you'll need to re-render this component every time.

Reply:
Looks like `getMenuProps` had the same problem, so this isn't making it worse.

Reply:
the styles can change if certain props change, so i guess we could cache the styles object and update it when props change. seems gross. another option could be to make the entire styles object static, specifying variable values like root width and height on specific root classes and passing the correct root className in at each size?